### PR TITLE
🚧 create a cpp solution for clockwise rotating a matrix

### PR DESCRIPTION
**Pull request by pranshta**

### CODE REVIEW

File: cpp/Rotate_Matrix.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	{ 
+		for (int j=0; j<C; j++) 
+		cout << mat[i][j] << " "; 
+		cout << endl; 
+	} 
+} 
+
+
+int main() 
+{ int n,m;
```
Comment (joaramirezra) [inline cpp/Rotate_Matrix.cpp:70]: in general i like it , just a little more order in spaces, indent and syntax , but in general is ok 
example :
line 76-77:
for (int i=0; i<R; i++) 
	{ 
		for (int j=0; j<C; j++) 
line 85-86
int main() 
{ int n,m;

**Comment by pranshta** [comment]
@joaramirezra  I have updated after making certain changes in indentation and syntax.

File: cpp/Rotate_Matrix.cpp (hunk 1 above)
Comment (tasnimzotder) [reply cpp/Rotate_Matrix.cpp:70]: Thanks for your review

**Review by tasnimzotder — CHANGES_REQUESTED** [review]
image

The program should return the clockwise rotated matrix. That is -
```bash
7 4 1
8 5 2
9 6 3
```

**Pull request closed without merging**